subprojects/GOPATH/src/github.com/microsoft/hivedscheduler/pkg/api/types.go

```diff
@@ -96,15 +96,20 @@ type PodBindInfo struct {
 	// The node to bind
 	Node string `yaml:"node"`
 	// The GPUs to bind
-	GpuIsolation          []int32  `yaml:"gpuIsolation"`
-	CellChain             string   `yaml:"cellChain"`
-	PodCellLevel          int32    `yaml:"podCellLevel"`
-	PodCellGpuPlacement   Range    `yaml:"podCellGpuRange"`
-	GroupCellLevel        int32    `yaml:"groupCellLevel"`
-	GroupCellNodes        []string `yaml:"groupCellNodes"`
-	GroupCellGpuPlacement Range    `yaml:"groupCellGpuRange"`
-	VirtualCellLevel      int32    `yaml:"virtualCellLevel"`
-	VirtualCellIndex      int32    `yaml:"virtualCellIndex"`
+	GpuIsolation      []int32                   `yaml:"gpuIsolation"`
+	CellChain         string                    `yaml:"cellChain"`
+	GroupScheduleInfo []GroupMemberScheduleInfo `yaml:"groupScheduleInfo"`
+}
+
+type GroupMemberScheduleInfo struct {
+	GpuNumber     int32              `yaml:"gpuNumber"`
+	PodPlacements []PodPlacementInfo `yaml:"podPlacements"`
+}
```

Comment:
To align with above AffinityGroup/AffinityGroupMember, how about this?

 AffinityGroupBindInfo []AffinityGroupMemberBindInfo
}

type AffinityGroupMemberBindInfo struct {
 GpuNumber     int32
 PodPlacements []PodPlacementInfo
}

Reply:
Fixed. But why we store CellChain in each AffinityGroupMemberBindInfo? All members should share a common chain.

---
In reply to: [332493391](https://github.com/microsoft/pai/pull/3684#discussion_r332493391) [](ancestors = 332493391)

Reply:
OK, my mistake, let's keep CellChain         

---
In reply to: [334216816](https://github.com/microsoft/pai/pull/3684#discussion_r334216816) [](ancestors = 334216816,332493391)